src/parquet/file/file-serialize-test.cc

```diff
@@ -97,6 +102,18 @@ TEST_F(TestSerialize, SmallFile) {
   ASSERT_EQ(values, values_out);
 }
 
+TEST_F(TestSerialize, SmallFileUncompressed) {
+  FileSerializeTest(Compression::UNCOMPRESSED, node_);
```

Comment:
Presumably because I'm unfamiliar with GTest. How should it be passed to the function?